src/java.base/share/classes/java/nio/charset/Charset.java

```diff
@@ -954,15 +954,15 @@ public final ByteBuffer encode(String str) {
      * @return A negative integer, zero, or a positive integer as this charset
      *         is less than, equal to, or greater than the specified charset
      */
+    @Override
```

Comment:
This PR adds `@Override` to a subset of the overridden methods, probably best to add it to all overridden methods so we don't end up with a mix.